src/core.js

```diff
@@ -420,7 +418,7 @@ $.extend( $.validator, {
 
 				result = this.check( checkElement ) !== false;
 				if ( result ) {
-					delete this.invalid[ checkElement.name ];
+					this.invalid[ checkElement.name ] = false;
```

Comment:
Instead of deleting this property, I assign to it `false` to refer to valid element. So:
`false` => valid element
`true` => invalid element